locations/spiders/sportsdirect.py

```diff
@@ -0,0 +1,69 @@
+import json
+import re
+
+import scrapy
+
+from locations.dict_parser import DictParser
+from locations.hours import DAYS, OpeningHours
+
+
+class SportsDirectSpider(scrapy.Spider):
+    name = "sportsdirect"
+
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+    available_countries = [
+        "MY",
+        "FR",
+        "AT",
+        "CY",
+        "DE",
+        "EE",
+        "KW",
+        "IE",
+        "LU",
+        "PL",
+        "SI",
+        "CZ",
+        "GB",
+        "AE",
+        "IS",
+        "HU",
+        "ES",
+        "LV",
+        "SK",
+        "BE",
+        "LT",
+        "PT",
+        "SA",
+        "RW",
+    ]
+    brand_mapping = {
+        "Sports Direct": {"brand": "Sports Direct", "brand_wikidata": "Q7579661"},
+        "Lillywhites": {"brand": "Lillywhites", "brand_wikidata": "Q6548397"},
+        "Field \u0026 Trek": {"brand": "Field & Trek"},
+    }
+
```

Comment:
Since it's not working on the server, lets flag it as needing a proxy:
 
```suggestion
    requires_proxy = True
```